### ISSUE TITLE
Change number of epochs for training

### ISSUE THREAD

**Issue by matlarsson** [jiny2001/dcscn-super-resolution#49]
When I try to train on my own data, the training always stops after 63 epochs. Is it possible to change it to another number of epochs?

**Comment by jiny2001** [comment]
Hi, try --lr_decay_epoch option. This is the number of training epochs for each learning rate and the default is 9.
